crypto/mem_dbg.c

```diff
@@ -204,29 +192,32 @@ static void app_info_free(APP_INFO *inf)
         OPENSSL_free(inf);
     }
 }
+#endif
 
 int CRYPTO_mem_ctrl(int mode)
 {
+#ifndef CRYPTO_MDEBUG
+    return mode - mode;
+#else
     int ret = mh_mode;
```

Comment:
There are no early returns and no accesses to this variable before the "return (ret);" at the end of the function; perhaps it should be removed.